src/textual/dom.py

```diff
@@ -62,7 +62,7 @@
 
 from typing_extensions import Literal
 
-_re_identifier = re.compile(IDENTIFIER)
+_re_identifier = re.compile(f"^{IDENTIFIER}$")
```

Comment:
Do we need to do this, or can we use `re.fullmatch` ?

Reply:
Tests pass with `re.fullmatch` instead of the formatted string identifier.
Do you want me to update the PR with `fullmatch`?

Reply:
Simpler perhaps. Go with full match.

Reply:
See 98e28f0.